extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaImageInsertDialog.js

```diff
@@ -0,0 +1,230 @@
+/*!
```

Comment:
Static properties (in ```.static.```) are inherited as well. However regardless of that I would like to encourage you to try to find better solution when you find yourself copying & pasting code - for instance method ```formatPolicy``` which is exactly the same in both cases.
I understand that methods ```insertPermanentMediaCallback``` and ```initialize``` are very similar but not exactly same - but it is hard to say from looking at this diff. If there is an alternation to default behavior required and currently that behavior is implemented somewhere within ```intialize``` for instance then I recommend extracting that logic (essentially splitting ```initialize``` in the base class to few smaller methods) and then overriding those small methods.
This way you not only achieve what you need to the scope of your current ticket but also improve overall quality of our codebase (most likely)